I've been editing the Pset student document (#3)
jen-reeve/Mineral-Stability: weathering_pset_student_1.Rmd

Mineral Stability Diagrams tell us what minerals are stable and what are not given a specific water chemistry. When a mineral is unstable it is a __reactant_ that will _weather_ into a _product_. 
The first step in using a mineral stability digram is making said mineral stability digram. The lines on the mineral stability digram describe regions where the two minerals are in equilibrium. 
\fra

> The x-axis of the diagram is $log[H_{4}SiO_{4}]$, which is silicic acid. The Y-axis in this case will be $log \frac {[K^{+}]}{[H^{+}]}$. While the X-axis will always be silicic acid, the Y-axis can have different cations depending on the mineral system of interest.

Author: True, thanks for that clarification!!

> You should have already derived all the weathering reactions that are relevant for the phase diagram we are building. For your reference, they are listed below.

Author: Got it. I'll go ahead and note that to fix.

> $2 KAlSi_{3}O_{8} + 9 H_{2}O + 2 H^{+} \leftrightharpoons Al_{2}Si_{2}O_{5}(OH)_{4} + 2 K^{+} + 4H_{4}SiO_{4},    Keq = 10^{-5.21}$ 

Author: Will format with Keq as subscript. and Yes, I'll number the equations and get them in order

$KAlSi_{3}O_{8} + H^{+} + 7H_{2}O \leftrightharpoons Al(OH)_{3} + 2K^{+} + 3H_{4}SiO_{4}. Keq = $
$KAlSi_{3}O_{8} + 2Al(OH)_{3} + H^{+}  \leftrightharpoons 1.5Al_{2}Si_{2}O_{5}(OH)_{4} + K^{+} + 0.5H_{2}O, Keq = $
$2KAlSi_{3}O_{8} + 2H^{+} + 9H_{2}O \leftrightharpoons Al_{2}Si_{2}O_{5}(OH)_{4} + 2K^{+} + 4H_{4}SiO_{4}, Keq = $

> **Question 1**: Using these equations and the Keq provided, write out the equations that will make up the mineral phase diagram. Remember that the lines are in the general linear form Y = mX + b. The x-axis of the diagram is $log[H_{4}SiO_{4}]$, which is silicic acid. The Y-axis in this case will be $log \frac {[K^{+}]}{[H^{+}]}$. Turn these equations in along with your work. *It is very helpful to remember your log rules when solving these equations.* 

Author: Will find a good link for log rules or a cheatsheet.

plot_k_spar_to_gibbs
```
**Question 2**: Look at the plot that was generated. Did pH increase or decrease? What about potassium? And silicic acid? Where did the potassium and silisic acid come from? What happened to the concentration of K-feldspar? What about Gibbsite?

> **Question 3**: Notice that as we are moving the water chemistry towards the K-feldspar field. However, we did not continue to react all of the K-feldspar into gibbsite. Why did we stop the reaction here?

Author: Will do.

plot_minerals_rxns_1_2
```

**Question 5** You have just plotted mineral abundance versus reaction progress. At what percent in the reaction does kaolinite start to form? Why is there an inverse relationship between kaolinite and gibbsite after this point? How does this relate to your phase diagram?

# Reaction 3: K feldspar weathering to kaolinite

> Now, all of our gibbsite has been reacted into kaolinite. However, we still have some K-feldspar left, and our water chemistry is now within the kaolinite stability field. So, our next weathering reaction we are concerned with is the K-feldspar/kaolinite system. 

Reviewer: ```suggestion
Now, all of our gibbsite has been reacted into kaolinite. However, K-feldspar is still not in equilibrium with the water water chemistry, and further reaction will move through the kaolinite stability field.
```
